src/settings/mod.rs

```diff
@@ -11,6 +11,23 @@ use deserialize;
 #[cfg(test)]
 mod test;
 
+#[derive(Debug, Deserialize)]
+pub struct AwsKeys
+{
+  #[serde(deserialize_with = "deserialize::aws_access")]
+  pub access: String,
+  #[serde(deserialize_with = "deserialize::aws_secret")]
+  pub secret: String,
```

Comment:
I ummed and ahhed about whether to add any validation for these two or not, in case I accidentally prohibit any valid keys. So they are validated, but it's pretty loose.

Reply:
On the plus side, prohibiting valid keys is something that will fail safely and be detected quickly, so this seems fine to me.